modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -151,14 +151,15 @@ spec:
 
 ----
 
-== Extending the core capabilities of Jenkins and OpenShift Pipelines by using plugins
-Jenkins has the advantage of a large ecosystem of numerous plugins developed over the years by its extensive user base. You can search and browse the plugins in the link:https://plugins.jenkins.io/[Jenkins Plugin Index].
+
+== Extending the core capabilities of Jenkins and Tekton by using plug-ins
+Jenkins has the advantage of a large ecosystem of numerous plug-ins developed over the years by its extensive user base. You can search and browse the plug-ins in the link:https://plugins.jenkins.io/[Jenkins Plug-in Index].
 
 OpenShift Pipelines also has many tasks developed and contributed by the community and enterprise users. A publicly available catalog of reusable OpenShift Pipelines tasks are available in the link:https://hub.tekton.dev/[Tekton Hub].
 
-In addition, OpenShift Pipelines incorporates many of the plugins of the Jenkins ecosystem within its core capabilities. For example, authorization is a critical function in both Jenkins and OpenShift Pipelines. While Jenkins ensures authorization using the link:https://plugins.jenkins.io/role-strategy/[Role-based Authorization Strategy] plugin, OpenShift Pipelines uses OpenShift's built-in Role-based Access Control system.
+In addition, Tekton incorporates many of the plug-ins of the Jenkins ecosystem within its core capabilities. For example, authorization is a critical function in both Jenkins and Tekton. While Jenkins ensures authorization using the link:https://plugins.jenkins.io/role-strategy/[Role-based Authorization Strategy] plug-in, Tekton uses OpenShift's built-in Role-based Access Control system.
```

Comment:
I didn't catch this in my initial review, but I believe you will need QA ack on this change.

Reply:
+1 to getting additional review.

Reply:
@opayne1 @kalexand-rh do we need QE ack when its a change to the content and not code

Reply:
@Amrita42, we need a QE ack any time it's a change that changes the meaning of the content. Code blocks are one of the areas that almost always requires QE ack, but body text isn't exempt.

Reply:
@VeereshAradhya please check this content and let me know if I change **plugin to plug-in** , does it change the meaning of the content.
The scope of this bz is to as per the IBM style guide **plug-in** is the correct word to use . 
I need a QE wrt to this specific content. Thanks.